test/core/security/tls_security_connector_test.cc

```diff
@@ -737,6 +737,44 @@ TEST_F(TlsSecurityConnectorTest,
                             on_peer_checked);
 }
 
+TEST_F(TlsSecurityConnectorTest,
+       ChannelSecurityConnectorWithVerifiedRootCertSubjectSucceeds) {
+  auto* sync_verifier_ = new SyncExternalVerifier(true);
```

Comment:
Why do we need to use `new` here?

Can we just write:
```
SyncExternalVerifier sync_verifier(true);
```

Reply:
This pattern copies the structure used in other tests - would we rather keep it matching all of the other tests or change it to this?

Reply:
Since we don't see a explicit delete or free here. My intuition is the ownership of this is taken somewhere below. If you prepared this on a stack, it would be freed twice, which is not good.